src/index.html

```diff
@@ -1,15 +1,194 @@
+          >
+        </label>
+      </fieldset>
+
+      <fieldset class="fieldset-section">
+        <legend>Registration</legend>
+
+        <label for="e-mail" class="form-field">
+          E-mail:
+
+          <input
+            type="email"
+            name="e-mail"
+            placeholder="email@example.com"
+            required
+            id="e-mail"
+          >
+        </label>
+
+        <label for="pass" class="form-field">
+          Password:
+
+          <input
+            type="password"
+            name="pass"
+            required
+            id="pass"
+          >
+        </label>
+      </fieldset>
+
+      <fieldset class="fieldset-section">
+        <legend>An interesting fact about you!</legend>
+
+        <div class="form-field">
+          Do you love cats?
+
+          <input
+            type="radio"
+            name="love-cats"
+            id="love-cats-yes"
+          >
+          <label for="love-cats-yes">Yes</label>
+
+          <input
+            type="radio"
+            name="love-cats"
+            id="love-cats-no"
+          >
+          <label for="love-cats-no">No</label>
+        </div>
+
+        <label for="love-color" class="form-field">
+          What is your favorite color?
+
+          <input
+            type="color"
+            name="love-color"
+            id="love-color"
+          >
+        </label>
+
+        <label for="time-to-bed" class="form-field">
+          What time do you go to bed?
+
+          <input
+            type="time"
+            name="time-to-bed"
+            id="time-to-bed"
+          >
+        </label>
+
+        <label for="car-brand" class="form-field">
+          What are your favorite brand of cars?
+
+          <select
+            name="car-brand"
+            multiple
+            id="car-brand"
+          >
+            <option value="BMW">BMW</option>
+            <option value="Audi">Audi</option>
+            <option value="Lada">Lada</option>
+          </select>
+        </label>
+
+        <label for="rate-work" class="form-field">
+          How do you rate our work?
+
+          <input
+            type="range"
+            name="rate-work"
+            min="1"
+            max="10"
+            value="1"
+            id="rate-work"
+          >
+        </label>
+      </fieldset>
+
+      <fieldset class="fieldset-section">
+        <legend>Additional info:</legend>
+
+        <label for="comments" class="form-field">
+          Comments:
+
+          <textarea
+            name="comments"
+            rows="2"
+            id="comments"
+         ></textarea>
+        </label>
+
+        <label for="recommend-us" class="form-field">
+          Would you recommend us?
+
+          <select name="recommend-us" id="recommend-us">
+            <option value="yes">Yes</option>
+            <option value="no">No</option>
+          </select>
+        </label>
+      </fieldset>
+
+      <button type="submit">Submit</button>
```

Comment:
type="submit" is a default type for buttons
https://developer.mozilla.org/en-US/docs/Web/HTML/Element/button

```suggestion
      <button>Submit</button>
```